static/app/components/gridEditable/styles.tsx

```diff
@@ -6,7 +6,7 @@ import {space} from 'sentry/styles/space';
 
 export const GRID_HEAD_ROW_HEIGHT = 45;
 export const GRID_BODY_ROW_HEIGHT = 40;
-export const GRID_STATUS_MESSAGE_HEIGHT = GRID_BODY_ROW_HEIGHT * 4;
+export const GRID_STATUS_MESSAGE_HEIGHT = GRID_BODY_ROW_HEIGHT * 3;
```

Comment:
we could avoid this change by fetching 4 rows of selectors rather than 1

Reply:
Does this impact other places around the app that use gridEditable?

Reply:
im ok with 4 rows :D

Reply:
yes but they look the same as the screenshots above:

<img width="892" alt="SCR-20230925-jnmz" src="https://github.com/getsentry/sentry/assets/56095982/af5b07b1-e98e-48f2-a09a-e277cca4f11e">

<img width="1245" alt="SCR-20230925-jnuy" src="https://github.com/getsentry/sentry/assets/56095982/d9122fcf-30e1-4964-bf54-657808e99e1c">

Reply:
gotcha let me change to 4 rows